src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceActions.tsx

```diff
@@ -23,6 +23,7 @@ import {
   KebabToggle,
 } from '@patternfly/react-core';
 import { removeInterface } from '@virtualmachines/actions/actions';
+import { isVMRunning } from '@virtualmachines/utils';
```

Comment:
ERROR in src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceActions.tsx:26:10
TS2305: Module '"@virtualmachines/utils"' has no exported member 'isVMRunning'.

Reply:
dint know why it didnt commit the second file, thanks